entities/backup/descriptor_test.go

```diff
@@ -473,3 +473,53 @@ func TestShardDescriptorClear(t *testing.T) {
 	s.ClearTemporary()
 	assert.Equal(t, want, s)
 }
+
+func TestBackupDescriptorValidate(t *testing.T) {
```

Comment:
Correct me if I'm wrong, but this doesn't appear to test the wildcard functionality? Also please include an acceptance for this, so we can verify it works end2end.

If you add a test [here](https://github.com/weaviate/weaviate/blob/main/test/helper/journey/backup_journey.go) for example, you can include it in the [backup journey tests](https://github.com/weaviate/weaviate/blob/main/test/helper/journey/backup_journey_tests.go#L22). The existing tests should give you a good idea of how to hook it in properly.

Reply:
Updating the tests for wildcard functionality